src/sdk/Diagnostics.ts

```diff
@@ -15,10 +15,22 @@ export class Diagnostics {
     private static privListener: ConsoleLoggingListener = undefined;
 
     public static SetLoggingLevel(logLevel: LogLevel): void {
-        this.privListener =  new ConsoleLoggingListener(logLevel);
+        this.privListener = new ConsoleLoggingListener(logLevel);
         Events.instance.attachConsoleListener(this.privListener);
     }
 
+    public static EnableConsoleOutput(): void {
```

Comment:
IIRC the native code loggers for speech and vision both went with start / stop paradigms.